editions/tw5.com/tiddlers/wikitext/Wikitext Cheat-Sheet.tid

```diff
@@ -0,0 +1,143 @@
+created: 20220821184022101
+modified: 20221010130927409
+tags: WikiText
+title: Wikitext Cheat-Sheet
+type: text/vnd.tiddlywiki
+
+\define extLink(text tid) [ext[$text$|https://tiddlywiki.com/#$tid$]]
+\define link(text tid) [[$text$|$tid$]]
+
+\define exportFilter()
+[[$(me)$]] [tag[$(me)$]]
+\end
+
+\define exportMeButton(btnText:"Export Me & Tagged")
+\whitespace trim
+<$set name="me" value=<<currentTiddler>> >
+  <$tiddler tiddler="$:/core/templates/exporters/JsonFile">
+    <$button class="tc-btn-download">
+      <$action-sendmessage $message="tm-download-file" $param=<<currentTiddler>> exportFilter=<<exportFilter>> filename={{{ [<me>addsuffix{!!extension}] }}}/>
+      <$text text="""$btnText$"""/>
```

Comment:
I do not think we should add any new macros that use textual substitution. Here's there's clearly an alternative.